src/jdk.internal.opt/share/classes/module-info.java

```diff
@@ -30,5 +30,5 @@
  */
 module jdk.internal.opt {
     exports jdk.internal.joptsimple to jdk.jlink, jdk.jshell;
-    exports jdk.internal.opt to jdk.jartool, jdk.jlink, jdk.jpackage;
+    exports jdk.internal.opt to jdk.compiler, jdk.jartool, jdk.javadoc, jdk.jlink, jdk.jpackage;
```

Comment:
Nit: line break at each module is easier to read and avoid long line like:
   https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/module-info.java#L149

Reply:
Yes, that's much nicer. Will reflow all other export directives as well.